### DISCUSSION TITLE
Installation on Windows 11

### DISCUSSION THREAD

**Discussion by NKcellMX** [Shidell/OpenCaldera#14] [Q&A]
Hello, I got my hands on a AGA and I'm using a M15 R3 updated to W11, however I can't even install the Alienware Graphics Amplifier software nor driver without getting "This installation package is not compatible with this computer"

Any ideas on how to make it work?

Thanks in advance

**Comment by Shidell** [comment]
Skip it, none of it is necessary for the AGA to work. Just plug-in your GPU to the AGA, connect it to your M15 R3, and power on.
